README.md

```diff
@@ -100,4 +115,21 @@ scripts/console --db
 To run migrations on the development database, use
 ```bash
 scripts/migrate
-```
\ No newline at end of file
+```
+
+To stage code and configurations and create template migrations for a version release, use
+```bash
+scripts/stageversion
+```
+
+## Release Process
+1) Make sure all your code is added and committed
+2) Create a PR against the main branch
+3) Once the PR has been merged, start the release process.
+4) Use `scripts/stagerelease` as documented in migrations section above making sure to rename any files ending in ".staged" in the migrations section
+5) Add details for release to the CHANGELOG
```

Comment:
This creates CHANGELOG entries that are not in `main`, since all the  changes from steps from the `scripts/stagerelease` are push directly to a tag.

I think it would be easier to keep track of things if everything flowed into `main`, and then `main` was tagged (or we had release branches that could be merged into and tagged). That's how it's worked on other projects.

One benefit of having everything in main first would be that all of the changes scripts/stagerelease could bring would be reviewable in a PR before tagging. In the method outlined here, there's not a chance to review local changes before pushing up and publishing them.

I'm OK with trying it this way, but I'm certainly more used to having tags be no new commits, and purely off of either main or release branches that have all the changes reviewable.